packages/mobile/src/screens/favorites-screen/FavoritesScreen.tsx

```diff
@@ -48,10 +56,38 @@ const favoritesScreens = [
 
 export const FavoritesScreen = () => {
   usePopToTopOnDrawerOpen()
+  const dispatch = useDispatch()
   const isOfflineModeEnabled = useIsOfflineModeEnabled()
 
   const savedTracks = useProxySelector(getTracks, [])
 
+  const handleFetchSavedPlaylists = useCallback(() => {
```

Comment:
Are these `useCallback`s valuable? Seems like we could just define the function and pass it to `useEffectOnce`